demos/python_demos/clip/README.md

```diff
@@ -0,0 +1,83 @@
+# CLIP demo with python node {#ovms_demo_clip}
```

Comment:
Can we extend the name of demo catalog to something like:
`clip_image_classification`, so it's more descriptive?